diff --git a/node_modules/react-native-web/dist/cjs/modules/createDOMProps/index.js b/node_modules/react-native-web/dist/cjs/modules/createDOMProps/index.js
index d643013..11ee0d3 100644
--- a/node_modules/react-native-web/dist/cjs/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/dist/cjs/modules/createDOMProps/index.js
@@ -806,7 +806,6 @@ var createDOMProps = (elementType, props, options) => {
 
   // Resolve styles
   if (pointerEvents != null) {
-    (0, _warnOnce.warnOnce)('pointerEvents', "props.pointerEvents is deprecated. Use style.pointerEvents");
   }
   var _StyleSheet = (0, _StyleSheet2.default)([style, pointerEvents && pointerEventsStyles[pointerEvents]], (0, _objectSpread2.default)({
       writingDirection: 'ltr'
diff --git a/node_modules/react-native-web/dist/modules/createDOMProps/index.js b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
index 18f5e8b..7662cc5 100644
--- a/node_modules/react-native-web/dist/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
@@ -802,7 +802,6 @@ var createDOMProps = (elementType, props, options) => {
 
   // Resolve styles
   if (pointerEvents != null) {
-    warnOnce('pointerEvents', "props.pointerEvents is deprecated. Use style.pointerEvents");
   }
   var _StyleSheet = StyleSheet([style, pointerEvents && pointerEventsStyles[pointerEvents]], _objectSpread({
       writingDirection: 'ltr'
diff --git a/node_modules/react-native-web/src/modules/createDOMProps/index.js b/node_modules/react-native-web/src/modules/createDOMProps/index.js
index 4144dbc..a051e92 100644
--- a/node_modules/react-native-web/src/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/src/modules/createDOMProps/index.js
@@ -870,14 +870,7 @@ const createDOMProps = (elementType, props, options) => {
         domProps.tabIndex = '0';
       }
     }
-  }
-
-  // Resolve styles
   if (pointerEvents != null) {
-    warnOnce(
-      'pointerEvents',
-      `props.pointerEvents is deprecated. Use style.pointerEvents`
-    );
   }
 
   const [className, inlineStyle] = StyleSheet(
